package.json

```diff
@@ -24,13 +24,23 @@
   "dependencies": {
     "@codecharacter-2022/map-designer": "workspace:packages/map-designer",
     "@codecharacter-2022/renderer": "workspace:packages/renderer",
+    "@fortawesome/fontawesome-svg-core": "^1.2.36",
+    "@fortawesome/free-brands-svg-icons": "^5.15.4",
+    "@fortawesome/free-solid-svg-icons": "^5.15.4",
+    "@fortawesome/react-fontawesome": "^0.1.16",
```

Comment:
Are all these icons being used?

Reply:
Yes, these icons are used. google icon uses brand-svg-icons and other icons in register page uses solid-svg-icons